src/ovi/Provider.php

```diff
@@ -0,0 +1,468 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Upmind\ProvisionProviders\SharedHosting\ovi;
```

Comment:
For consistency's sake can you please rename the directory + namespace from `ovi` to `OviPanel`?